src/mainAppBar/userProfile.component.tsx

```diff
@@ -108,15 +115,21 @@ const UserProfileComponent = (
               </Typography>
             </div>
             <Divider />
-            <MenuItem onClick={logout}>
+            <MenuItem aria-label="Manage cookies item" onClick={manageCookies}>
```

Comment:
I'm not sure these strictly need aria labels, since a screen reader will be able to read the text associated with the actions

Reply:
@louise-davies I have removed the aria-labels from both menu-items now.